build-support/bin/generate_travis_yml.py

```diff
@@ -543,7 +546,7 @@ def integration_tests_v2(python_version: PythonVersion) -> Dict:
     "name": f"Integration tests - V2 (Python {python_version.decimal})",
     "script": [
       (
-        "travis_wait 65 ./build-support/bin/ci.py --integration-tests-v2 "
+        "travis-wait-enhanced --timeout 65m -- ./build-support/bin/ci.py --integration-tests-v2 "
```

Comment:
I'd prefer that we make this a precursor PR. It's a great improvement, although not related to speculation.

Reply:
Ok I'll split this change out today.

Reply:
PR here: https://github.com/pantsbuild/pants/pull/8446